powerauth-webflow-authentication-form/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/method/form/controller/FormLoginController.java

```diff
@@ -121,7 +121,8 @@ protected AuthenticationResult authenticate(UsernamePasswordAuthRequest request)
             FormData formData = new FormDataConverter().fromOperationFormData(operation.getFormData());
             ApplicationContext applicationContext = operation.getApplicationContext();
             OperationContext operationContext = new OperationContext(operation.getOperationId(), operation.getOperationName(), operation.getOperationData(), formData, applicationContext);
-            ObjectResponse<UserDetailResponse> lookupResponse = dataAdapterClient.lookupUser(request.getUsername(), request.getOrganizationId(), operationContext);
+            // Client certificate is not yet supported in non SCA login method
+            ObjectResponse<UserDetailResponse> lookupResponse = dataAdapterClient.lookupUser(request.getUsername(), request.getOrganizationId(), null, operationContext);
```

Comment:
we should avoid passing any null value this way :)

Reply:
The `clientCertificate` is always `null` because the legacy form login does not support certificates (and probably never will, we want to deprecate these methods after all countries use SCA, even outside EU). I will put the `null` value into a variable to highlight this.

Reply:
ok, sounds good